.nvmrc

```diff
@@ -0,0 +1 @@
+v20.9.0
```

Comment:
@frikit could you update either in the ticket or in this PR, regarding the reason you have updating to this version (node 20.9.0) instead of node 20.18.0.